frontend/src/reports/TwoVariableReport.tsx

```diff
@@ -211,6 +211,7 @@ function TwoVariableReport(props: {
           updateFips: (fips: Fips) => void
         ) => (
           <UnknownsMapCard
+            overrideAndWithOr={true}
```

Comment:
do we need to make sure this is only true in the case of race_and_ethnicity as well?

Reply:
Yes, good catch. I'll fix that